src/Goerli-DssSpell.sol

```diff
@@ -55,52 +54,40 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
         // onboardNewCollaterals();
+        // offboardCollaterals();
 
-        // ----------------------------- MKR Vesting -----------------------------
+        // ---------------------- CU DAI Vesting Streams -----------------------
         // NOTE: ignore in goerli
 
-        // ------------------------ Delegate Compensation ------------------------
+        // ---------------------- SPF Funding Transfers ------------------------
         // NOTE: ignore in goerli
 
-        // ------------------ PPG - Maker Open Market Committee ------------------
-        // https://vote.makerdao.com/polling/QmXHnn2u#poll-detail
-
-        ////// Stability Fee Changes //////
-
-        // Decrease the CRVV1ETHSTETH-A Stability Fee from 2.25% to 2.0%.
+        // ------------------- GRO-001 MKR Stream Clean-up ---------------------
         // NOTE: ignore in goerli
-        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", TWO_PCT_RATE, true);
-
-        // Decrease the MANA-A Stability Fee from 6% to 4.5%.
-        DssExecLib.setIlkStabilityFee("MANA-A", FOUR_FIVE_PCT_RATE, true);
 
-        // Decrease the ETH-A Stability Fee from 2.25% to 1.5%.
-        DssExecLib.setIlkStabilityFee("ETH-A", ONE_FIVE_PCT_RATE, true);
+        // -------------------- Update HVB Legal Documents ---------------------
+        bytes32 ilk                      = "RWA009-A";
+        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress(
+            "MIP21_LIQUIDATION_ORACLE"
+        );
 
-        // Decrease the ETH-B Stability Fee from 3.75% to 3.0%.
-        DssExecLib.setIlkStabilityFee("ETH-B", THREE_PCT_RATE, true);
+        ( , address pip, uint48 tau, ) = RwaLiquidationLike(
```

Comment:
Nit - also this might be nicer as a one liner.

Reply:
This is my identity.  Are you denying my experience as an aging neckbeard that used monochrome terminals?  REEEEEE!!

Reply:
Please don't shout at me.